board.py

```diff
@@ -78,7 +78,7 @@ def disp_board(self):
         print()
         for i in range(self.width):
             print(i + 1,end=' ')
-
+    # does not print a newline at the end
```

Comment:
This means, when you call disp_board() then print something else, the next print shows up on the same line as your column numbers.